jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -15,21 +16,37 @@
 
 lock = locks.Lock()
 MOUNT_SERVER_PORT = 9002
+HTTP_UNIX_SOCKET_SCHEMA="http_unix"
+HTTP_SCHEMA="http"
 
+DEFAULT_SCHEMA=HTTP_UNIX_SOCKET_SCHEMA
```

Comment:
How is the schema intended to be configured? It appears to be that this isn't the "default" it's the selected schema and there's no way to switch between the two settings.

Reply:
I could pass it from the env.py file. That file would then need to be updated if we wanted to switch the schema.

Reply:
It is now coming from env.py